pkg/server/status/runtime.go

```diff
@@ -115,6 +115,12 @@ var (
 		Measurement: "RSS",
 		Unit:        metric.Unit_BYTES,
 	}
+	metaRSSPct = metric.Metadata{
```

Comment:
While you're here, could you add a `sys.system_total` (or something similar) to export the total amount of memory in the system?

Reply:
uh. or `sys.mem.system`.